EAP-TLS_Solution/EAP-TLS Client/README.md

```diff
@@ -0,0 +1,175 @@
+    |`const char *const g_eapTlsClientPrivateKeyRelativePath = "certs/iotuser_private.pem";`|Change `certs/iotuser_private.pem` to reflect relative path and file name of the *private key* of the client certificate.  **Note**: this would normally be returned by the WebAPI and will be stored in Mutable storage as a DeviceConfiguration object, therefore initializing it would be for testing purposes only.|
+     |`const char *const g_eapTlsClientPrivateKeyPassword = "<optional private key password>";`|If  `USE_CLIENT_CERT_PRIVATE_KEY_PASS_FROM_WEBAPI` is **not** defined, set this value to reflect the password to be used for decrypting the client certificate's private key. |
+    #
+    **NOTE**: all the above certificate names refer to the sample certificates described in the in [Prerequisites](#Prerequisites) section above.
+
+5. In [tests\test.c](.\tests\test.c), modify the following constant definitions to reflect your network/certificates setup, as described in the [Prerequisites](#Prerequisites) section above:
+
+    |Variable|Purpose|
+    |-|-|
+    |`const char *const g_eapTlsInvalidRootCaCertificateRelativePath = "certs/bad-az-CA.pem";`|Change `certs/bad-az-CA.pem` to reflect relative path and file name of the *public* part of a RootCA certificate, with which the client will **not** be able to be authenticated by the RADIUS server, which will be used to test an authorization failure.|
+    |`const char *const g_eapTlsInvalidClientIdentity = "extuser@azsphere.com";`|Change `extuser@azsphere.com` to a client identity that is the one to which the `extuser_public.pem` and `certs/extuser_private.pem` were issued to, which will be used to test an authorization failure. |
+    |`const char *const g_ eapTlsInvalidClientCertificateRelativePath = "certs/extuser_public.pem";`|Change `certs/extuser_public.pem` to reflect relative path and file name of the *public* part of the client certificate, which will be used to test an authorization failure. |
+    |`const char *const g_eapTlsInvalidClientPrivateKeyRelativePath = "certs/extuser_private.pem";`|Change `certs/extuser_private.pem` to reflect relative path and file name of the *private key* of the client certificate, which will be used to test an authorization failure. |
+    #
+
+
+### Using the client EAP-TLS library standalone
+
+In the [tests.c](.\tests\tests.c) source file, the `TestEapTlsLib_InitializeConfiguration` function shows how to set up the *bootstrap network* (over Wi-Fi or Ethernet) and how to fully initialize a local `struct` variable of type `EapTlsConfig`, with which many of the `EapTls_*` APIs work with.
```

Comment:
Shouldn't this be "test.c", not "tests.c"?  The folder is "tests" but the file is "test.c", is that right?